pennylane/transforms/core/transform_dispatcher.py

```diff
@@ -92,6 +96,54 @@ def is_informative(self):
         """Return True is the transform does not need to be executed."""
         return self._is_informative
 
+    def custom_qnode_transform(self, fn):
```

Comment:
Wasn't this supposed to be the role of "classical cotransform"?

Reply:
Not as I understand it, but tagging @rmoyard

Reply:
No it is separated, classical co transform are function you need during the post processing (classical jacobian mainly)